modules/candidate_profile/test/candidate_profileTest.php

```diff
@@ -97,7 +97,7 @@ function testCandidateProfileWithoutProjectPermissions()
      */
     function testCandidateProfileInstrumentLink1()
     {
-        $this->safeGet($this->url . "/candidate_profile/900000/");
+        $this->safeGet($this->url . "/candidate_profile/300001/");
         $this->safeFindElement(
             WebDriverBy::cssSelector(
                 "#card2 > div:nth-child(1) >".
```

Comment:
#card2 > div > div > div > div:nth-child(1) > div > div:nth-child(2) > h4 > a

Reply:
$this->safeFindElement(
            WebDriverBy::cssSelector(
                "#card2 > div > div > div > div:nth-child(1) > div > div:nth-child(2) > h4 > a"
            )
        )->click();